 Don't FC on T9 search input selection of default setting.

* This does NOT help in selecting of default option.
* Needs to debug reason of FC.

diff --git a/src/com/android/dialer/settings/DisplayOptionsSettingsFragment.java b/src/com/android/dialer/settings/DisplayOptionsSettingsFragment.java
index 816e1ab..d80ed90 100644
--- a/src/com/android/dialer/settings/DisplayOptionsSettingsFragment.java
+++ b/src/com/android/dialer/settings/DisplayOptionsSettingsFragment.java
@@ -66,7 +66,11 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public boolean onPreferenceChange(Preference preference, Object objValue) {
         if (preference == mT9SearchInputLocale) {
-            saveT9SearchInputLocale(preference, (String) objValue);
+            try {
+                saveT9SearchInputLocale(preference, (String) objValue);
+            } catch(IllegalArgumentException e) {
+                // Do nothing
+            }
         }
         return true;
     }
